### ISSUE TITLE
ARM-TTK: Tests are run against all JSON files in a folder when a full file path is provided

### ISSUE THREAD

**Issue by sam-cogan** [Azure/azure-quickstart-templates#7132]
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Sorry to hear you had a bad experience with one of the templates :worried: But, in case you're just asking a question, we're happy to help. You can also check if the question might already have been asked here https://github.com/Azure/azure-quickstart-templates/issues?utf8=%E2%9C%93&q=is%3Aissue

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

[Template Name goes here](Template link goes here)

### Issue Details
When running the ARM TTK and supplying a full path to a json file for the TemplatePath parameter I would expect tests just to run against that specific file. However, if there are other JSON files in that folder, then tests will be run against them anyway.
### Repro steps (*if necessary, delete otherwise*)
1. Create a folder with multiple ARM templates in
2. Run Test-AzTemplate specifying the full path to one of these files for the TemplatePath parameter
3. See tests run against all files

**Comment by bmoore-msft** [comment]
The -File param is meant for a single file test, but that's not working at the moment either...

**Comment by sam-cogan** [comment]
Yeah I managed to get it working by using a folder in "TemplatePath" and then the file, or filter in the "File" parameter
